src/ClusterBootstrap/services/monitor/grafana-config.yaml

```diff
@@ -0,0 +1,157 @@
+{% set prometheus_url = "192.168.255.1:9091" %}
```

Comment:
How about "{{ cnf["api-server-ip"] }}:9091" ?

Reply:
This assumes api-server is always co-allocated with prometheus, we can assume that?

Reply:
Have tried this, the value will be `10.3.0.1` in dltsp40 cluster, the grafana pod can not connect to this address, not sure what this IP is.